web/packages/teleterm/package.json

```diff
@@ -44,8 +44,8 @@
     "@types/whatwg-url": "^11.0.5",
     "@xterm/xterm": "^5.5.0",
     "@xterm/addon-fit": "^0.10.0",
-    "electron": "31.1.0",
-    "electron-builder": "^25.0.1",
+    "electron": "31.3.1",
+    "electron-builder": "^25.0.3",
```

Comment:
I looked through everything (macOS arm64 and amd64, Windows on VM, real Ubuntu and Fedora in amd64 VM) but checking for changes in after install scripts and window focus (because of that annoying change where v17 clients are incompatible with v16 auth servers).

@gzdunek Could you look at changes in after install scripts tomorrow?

Reply:
Sure, I will do it.

Reply:
I tested the changes in the after install script and I didn't see any problems on Ubuntu 22.04.
Same for windows focus on all OSes.